packages/client/src/createAttachmentFromRid.ts

```diff
@@ -14,27 +14,24 @@
  * limitations under the License.
  */
 
-import type {
-  Attachment,
-  AttachmentMetadata,
-  AttachmentSignature,
-} from "@osdk/client.api";
+import type { Attachment } from "@osdk/client.api";
 import { Ontologies } from "@osdk/internal.foundry";
 import type { MinimalClient } from "./MinimalClientContext.js";
 
-export function createAttachmentReader(
+export function createAttachmentFromRid(
```

Comment:
This helper function can be used for people to reconstruct an attachment object with just a rid and client